sdk/php/dev/src/PhpSdkDev.php

```diff
@@ -28,6 +28,11 @@ public function test(
         );
     }
 
+    #[DaggerFunction('Run linter in source directory')]
+    public function lint(Directory $source): Container {
+        return $this->base($source)->withExec(['phpcs']);
+    }
```

Comment:
You need a "format" function as well. It'll make it easier for other maintainers that get PHP linting errors, to run the formatter in order to solve them.

You can return a Directory with the diff of the changes, like this:

https://github.com/dagger/dagger/blob/b0d27e15843341643954a4e9ef0841cd785b78b3/sdk/python/dev/src/main/__init__.py#L186-L187

Reply:
I've added #8124 to ensure I get that done in a follow up.

If we can get this merged separately the `format()` should be a fairly small PR.